core-api/src/test/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfigTest.java

```diff
@@ -0,0 +1,115 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+package com.optimizely.ab.optimizelyconfig;
+
+import com.optimizely.ab.Optimizely;
+import com.optimizely.ab.config.*;
+import org.junit.Before;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static com.optimizely.ab.config.DatafileProjectConfigTestUtils.*;
+
+public class OptimizelyConfigTest {
+
+    private OptimizelyConfig optimizelyConfig;
+    private Optimizely optimizely;
+    private ProjectConfig projectConfig;
+
+    @Before
+    public void initialize()throws Exception {
+        projectConfig = new DatafileProjectConfig.Builder().withDatafile(validConfigJsonV4()).build();
+        optimizely = Optimizely.builder().withDatafile(validConfigJsonV4()).build();
+        optimizelyConfig = optimizely.getOptimizelyConfig();
```

Comment:
As a unit test, we shouldn't have a dependency on `Optimizely`. (It's also not needed as a class member anyway) 
We should be testing the service directly.
```java
projectConfig = new DatafileProjectConfig.Builder().withDatafile(validConfigJsonV4()).build();
optimizelyConfig = new OptimizelyConfigService(projectConfig).getOptimizelyConfig();
```